config/table-headers.js

```diff
@@ -683,3 +691,40 @@ export const CONFIGURED_RECEIVER = {
   formatter:     'Link',
   formatterOpts: { options: { internal: true } },
 };
+
+export const ACCESS_KEY = {
+  name:     'name',
+  labelKey: 'tableHeaders.accessKey',
+  align:    'left',
+  sort:     ['name'],
+  width:    200,
+};
+
+export const SCOPE = {
+  name:        'scope',
+  value:       'clusterName',
+  labelKey:    'tableHeaders.scope',
+  dashIfEmpty: true,
+  align:       'left',
+  sort:        ['scope'],
+  width:       100,
+};
+
+export const EXPIRES_AT = {
+  name:          'expiresAt',
+  value:         'expiresAt',
+  labelKey:      'tableHeaders.expiresAt',
+  align:         'left',
+  width:         200,
+  formatter:     'Date',
```

Comment:
LiveDate

Reply:
I used date to be consistent with the Ember UI - I can change to LiveDate - I'll change the column name to be 'Expires' rather than 'Expires At'.

Reply:
I changed this back to Date - otherwise if the token expires when you are viewing the page, the live date starts counting back up. I updated the state column to correctly show the expired state.

Reply:
This should have a `sort: ['expiresAt']` to make it sortable.

Formatters are a shortcut but not the only way to customize a table cell's displayed content.. you can instead directly define template contents for it (in pages/account/index.vue), which can cover the empty, not expired, and expired cases separately.

```html
<ResourceTable...>
  <template #cell:expires="{row}">
    <span v-if="row.isExpired" class="text-error">
      <!-- assuming an isExpired is created to address the lack of recomputing thing mentioned elsewhere -->
      <LiveDate :value="row.expiresAt" :add-suffix="true" />
    </span>
    <template v-else-if="row.expiresAt">
      <LiveDate :value="row.expiresAt" :add-suffix="false" />
    </template>
    <span v-else v-t="'generic.never'" class="text-muted" />
  </template>
</ResourceTable>
```